<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Öğrenci Bilgilerini Alma Arayüzü</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Ad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Soyad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Bölüm:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_ad">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_soyad">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_egitim">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title">
    <string>Eğitim</string>
   </property>
   <widget class="QRadioButton" name="rd_lisans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisans</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_lisansustu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisansüstü</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_sinif">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Sınıf</string>
   </property>
   <widget class="QRadioButton" name="rd_sinif1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4.Sınıf</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_zorunluderslerlisans">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>209</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Alınan Zorunlu Dersler</string>
   </property>
   <widget class="QCheckBox" name="ch_Aiit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AİİT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_Turkdili">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Türk Dili</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_isg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>İSG</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_durum">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Durum</string>
   </property>
   <widget class="QRadioButton" name="rd_ders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ders</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_tez">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tez</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_zorunluderslerlisansustu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Alınan Zorunlu Dersler</string>
   </property>
   <widget class="QCheckBox" name="ch_secmeli">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçmeli Dersler</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_tez">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tez</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_seminer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seminer</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_bilgileriYazdir">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Bilgileri Yazdır</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mezuniyetSorgula">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Mezuniyet Sorgula</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ogrDurumYazdir">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Öğrenci Durum Yazdır</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_bolum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
